fe/fe-core/src/main/java/com/starrocks/sql/analyzer/AlterTableClauseVisitor.java

```diff
@@ -698,6 +701,48 @@ public Void visitDropColumnClause(DropColumnClause clause, ConnectContext contex
         return null;
     }
 
+    @Override
+    public Void visitAddFieldClause(AddFieldClause clause, ConnectContext context) {
+        String columnName = clause.getColName();
+        if (Strings.isNullOrEmpty(columnName)) {
+            throw new SemanticException(PARSER_ERROR_MSG.invalidColFormat(columnName));
+        }
+
+        if (!(table instanceof OlapTable)) {
```

Comment:
materialized view is not supported？

Reply:
> materialized view is not supported？

Not support right now. Should we support materialized view?

Reply:
better use `!table.isOlapTable()`